### DISCUSSION TITLE
Product overview and knowledge base

### DISCUSSION THREAD

**Discussion by ProgyNL** [Paymenter/Paymenter#241] [Ideas]
For now, customers can only see the products via the home page, but I would personally agree if others would also agree with it, of course, whether it was possible that, just like for invoices and tickets, there would also be a product overview page and knowledge base page for customers and via the administration panel can also manage the knowledge base

**Comment by finnjosten** [comment]
Make the product detail page on a toggle for each product as I don't want all my products to have that
